crates/sui-indexer/src/lib.rs

```diff
@@ -2,11 +2,13 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use std::net::{IpAddr, Ipv4Addr, SocketAddr};
+use std::str::FromStr;
```

Comment:
question: where is this used?

Reply:
here `pv4Addr::from_str`

Reply:
Ipv4Addr::from_str needs that to compile

Reply:
ah i see, i assumed that ipv4addr would have from_str. ig it's a rust thing haha, thank you